javascript/javascript1/week3/travel_time.js

```diff
@@ -0,0 +1,12 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function calculateTime(travelObject) {
```

Comment:
Do you think you will understand this code immediately if you come back to it in 5 months? Could any of the lines here use an explanatory comment of what is going on?

Reply:
comments are definitely helpful here!